src/solc_0.8/polygon/child/land/PolygonLandV2.sol

```diff
@@ -0,0 +1,164 @@
+     * @param to The recipient of the token
+     * @param id The id of the token
+     */
+    function transferFrom(
+        address from,
+        address to,
+        uint256 id
+    ) external override onlyAllowedOperator(from) {
+        _checkTransfer(from, to, id);
+        _transferFrom(from, to, id);
+        if (to.isContract() && _checkInterfaceWith10000Gas(to, ERC721_MANDATORY_RECEIVER)) {
+            require(_checkOnERC721Received(_msgSender(), from, to, id, ""), "ERC721_TRANSFER_REJECTED");
+        }
+    }
+
+    /**
+     * @notice Transfer a token between 2 addresses letting the receiver knows of the transfer
+     * @param from The sender of the token
+     * @param to The recipient of the token
+     * @param id The id of the token
+     * @param data Additional data
+     */
+    function safeTransferFrom(
+        address from,
+        address to,
+        uint256 id,
+        bytes memory data
+    ) public override onlyAllowedOperator(from) {
+        _checkTransfer(from, to, id);
+        _transferFrom(from, to, id);
+        if (to.isContract()) {
+            require(_checkOnERC721Received(_msgSender(), from, to, id, data), "ERC721_TRANSFER_REJECTED");
+        }
+    }
+
+    /**
+     * @notice Transfer a token between 2 addresses letting the receiver knows of the transfer
+     * @param from The send of the token
+     * @param to The recipient of the token
+     * @param id The id of the token
+     */
+    function safeTransferFrom(
+        address from,
+        address to,
+        uint256 id
+    ) external override onlyAllowedOperator(from) {
+        safeTransferFrom(from, to, id, "");
+    }
+
+    /**
+     * @notice Set the approval for an operator to manage all the tokens of the sender
+     * @param operator The address receiving the approval
+     * @param approved The determination of the approval
+     */
+    function setApprovalForAll(address operator, bool approved)
+        external
+        override
+        onlyAllowedOperatorApproval(operator)
+    {
+        _setApprovalForAll(_msgSender(), operator, approved);
+    }
+
+    /**
+     * @notice Set the approval for an operator to manage all the tokens of the sender
+     * @param sender The address giving the approval
+     * @param operator The address receiving the approval
+     * @param approved The determination of the approval
+     */
+    function setApprovalForAllFor(
+        address sender,
+        address operator,
+        bool approved
+    ) external override onlyAllowedOperatorApproval(operator) {
+        require(sender != address(0), "Invalid sender address");
+        address msgSender = _msgSender();
+        require(msgSender == sender || _superOperators[msgSender], "UNAUTHORIZED_APPROVE_FOR_ALL");
+
+        _setApprovalForAll(sender, operator, approved);
+    }
+
+    /// @notice This function is used to register Land on the Operator filterer Registry of Opensea.can only be called by admin.
```

Comment:
Fix typos
This function is used to register the Land _contract_ on the Operator filterer Registry of Opensea
@dev can only be called by admin.